src/dataclass_binder/_impl.py

```diff
@@ -697,6 +706,10 @@ def _to_toml_pair(value: object) -> tuple[str | None, Any]:
                     return "-weeks", days // 7
                 else:
                     return "-days", days
+        case Enum():
+            return None, value.name.lower()
+        case ReprEnum():
+            return None, value.value
```

Comment:
This line also has no test coverage, which I think may be a result of a bug: any `IntEnum` member inherits both `int` and `ReprEnum`, but `int` is checked in the very first `case` statement, so control never gets here.